Spigot-API-Patches/0227-Add-StructureLocateEvent.patch

```diff
@@ -0,0 +1,149 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: dfsek <dfsek@protonmail.com>
+Date: Tue, 15 Sep 2020 21:59:16 -0700
+Subject: [PATCH] Add StructureLocateEvent
+
+
+diff --git a/src/main/java/org/bukkit/event/world/StructureLocateEvent.java b/src/main/java/org/bukkit/event/world/StructureLocateEvent.java
+new file mode 100644
+index 0000000000000000000000000000000000000000..1011b03fcc7be912e617a6fde3f62e68d70f2c1c
+--- /dev/null
++++ b/src/main/java/org/bukkit/event/world/StructureLocateEvent.java
+@@ -0,0 +1,137 @@
++package org.bukkit.event.world;
```

Comment:
```suggestion
+package io.papermc.paper.event.world;
```

Reply:
Shouldn't this be `com.destroystokyo.paper`, or are you guys moving away from that package?

Reply:
Moving away